src/chainparams.h

```diff
@@ -113,4 +116,24 @@ CChainParams& Params(const std::string& chain);
  */
 void SelectParams(const std::string& chain);
 
+// bip135 begin
+/**
+ * Return the currently selected parameters. Can be changed by reading in
+ * some additional config files (e.g. CSV deployment data)
+ */
+CChainParams &ModifiableParams();
+
+/**
+ * Returns true if a deployment is considered active on a particular network
+ */
+
+bool isConfiguredDeployment(const Consensus::Params& consensusParams, const int bit);
+
+/**
+ * Dump the fork deployment parameters for the given BIP70 chain name.
```

Comment:
Looking at BIP70, it seems it is referring to the 'network' attribute definition which is there given as 

> either "main" for payments on the production Bitcoin network, or "test" for payments on test network. If a client receives a PaymentRequest for a network it does not support it must reject the request.

So I guess BIP70 might have been the first to formally introduce this 'network' concept with the values 'main' and 'testnet', even if earlier code already used it.